rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -1026,57 +1026,62 @@ where
                     vec![this_canister_id, canisters.root.unwrap()],
                 )
                 .await
-                .map_err(|e| {
+                .map_err(|err| {
                     format!(
-                        "Unable to set Root as Governance canister controller: {}",
-                        e
+                        "Unable to set SNS-W and Root as Governance canister controller: {}",
+                        err
                     )
                 }),
-            // Set root as controller of Ledger.
+            // Set Root as controller of Ledger.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.ledger.unwrap()),
                     vec![this_canister_id, canisters.root.unwrap()],
                 )
                 .await
-                .map_err(|e| format!("Unable to set Root as Ledger canister controller: {}", e)),
-            // Set root as controller of Index.
+                .map_err(|err| {
+                    format!(
+                        "Unable to set SNS-W and Root as Ledger canister controller: {}",
+                        err
+                    )
+                }),
+            // Set Root as controller of Index.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.index.unwrap()),
                     vec![this_canister_id, canisters.root.unwrap()],
                 )
                 .await
-                .map_err(|e| format!("Unable to set Root as Index canister controller: {}", e)),
+                .map_err(|err| {
+                    format!(
+                        "Unable to set SNS-W and Root as Index canister controller: {}",
+                        err
+                    )
+                }),
             // Set Governance as controller of Root.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.root.unwrap()),
                     vec![this_canister_id, canisters.governance.unwrap()],
                 )
                 .await
-                .map_err(|e| {
+                .map_err(|err| {
                     format!(
-                        "Unable to set Governance as Root canister controller: {}",
-                        e
+                        "Unable to set SNS-W and Governance as Root canister controller: {}",
+                        err
                     )
                 }),
-            // Set SNS Root and NNS Root as controllers of Swap.
+            // Set Root as the controller of Swap.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.swap.unwrap()),
-                    vec![
-                        this_canister_id,
-                        canisters.root.unwrap(),
-                        ROOT_CANISTER_ID.get(),
```

Comment:
NC.

This appears to be the operative change of this whole PR, yes?

The other changes here seem to be orthogonal minor improvements. It is not ideal for those to be here, because it _**buries the lede**_, and thus, degrades human understanding. Some ways to bring the lede (back) to the fore:

1. include a diff reading guide
2. do not include uninteresting changes (I already suggested this earlier)
3. shunt uninteresting changes to their own commits in the PR